webhook/tests/test_query_results.py

```diff
@@ -83,29 +80,26 @@ def test_query_results_links_and_meta():
     }
 
     r = client.post(
-        '/query-results/',
+        URL,
         headers=correct_headers,
-        json={"data": [], "links": links, "meta": meta})
+        json={"api_key": API_KEY, "data": [], "links": links, "meta": meta})
 
     assert r.status_code == 200
```

Comment:
You can also check that result was logged

Reply:
I think that we can trust that python logger is working.